lms/templates/courses_list.html

```diff
@@ -0,0 +1,27 @@
+<%namespace name='static' file='static_content.html'/>
+<%! from django.utils.translation import ugettext as _ %>
+
+<section class="courses-container">
+  <section class="highlighted-courses">
+
+    % if settings.FEATURES.get('COURSES_ARE_BROWSABLE'):
```

Comment:
I'd prefer that we do not reference settings directly in templates and instead provide switches/information via the context